packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -221,3 +221,50 @@ class ThemedColor {
     return colorScheme.isLight ? light : dark;
   }
 }
+
+// TODO(elliette): Add mixin for comparing enums. Eg, size > MediaSize.s
+// Requires enhanced enums feature in Dart SDK >= 2.17.
+enum MediaSize {
+  xxs,
+  xs,
+  s,
+  m,
+  l,
+  xl,
+}
+
+class ScreenSize {
```

Comment:
can you add tests for this class? you should be able to use the testWidgetsWithWindowSize helper to set screen sizes for the tests

Reply:
Done!